From eb52ed215971833cb382b9abc907669e868903b4 Mon Sep 17 00:00:00 2001
From: Biswapriyo Nath <nathbappai@gmail.com>
Date: Sun, 3 Apr 2022 14:24:04 +0530
Subject: [PATCH] Revert "Fix check for openssl on Windows"

This reverts commit 045e5c5a4460020e513516a5d1f3087094e67da3
In Windows platform, openssl libraries have same name
(libcrypto and libssl) like other unix-like platforms.
---
 configure.ac | 19 +------------------
 1 file changed, 1 insertion(+), 18 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2c9e1f957..7c6d577ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1209,24 +1209,7 @@ fi
 if test "x$with_openssl" != "xno"; then
     AC_CHECK_HEADERS([openssl/evp.h])
     saved_LIBS=$LIBS
-    case "$host_os" in
-      *mingw* | *cygwin* | *msys*)
-        case "$host_cpu" in
-          x86_64)
-            AC_CHECK_LIB(eay64,OPENSSL_config)
-            if test "x$ac_cv_lib_eay64_main" != "xyes"; then
-              AC_CHECK_LIB(eay32,OPENSSL_config)
-            fi
-            ;;
-          *)
-            AC_CHECK_LIB(eay32,OPENSSL_config)
-            ;;
-        esac
-        ;;
-      *)
-        AC_CHECK_LIB(crypto,OPENSSL_config)
-        ;;
-    esac
+    AC_CHECK_LIB(crypto,OPENSSL_config)
     CRYPTO_CHECK(MD5, OPENSSL, md5)
     CRYPTO_CHECK(RMD160, OPENSSL, rmd160)
     CRYPTO_CHECK(SHA1, OPENSSL, sha1)
